### ISSUE TITLE
Invalid icon for CRD resources in list view

### ISSUE THREAD

**Issue by olivierlemasle** [argoproj/argo-cd#2977]
Checklist:

* [x] I've searched in the docs and FAQ for my answer: http://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

In resource tree, we have nice icons for CRD resources, since #2653. However, this does not work in list view.

**To Reproduce**

Create an application with custom resources. Look at the icons for these custom resources **in list view**.

**Expected behavior**

It is expected to have the same icons than in resource tree (the resource's kind initials).

**Screenshots**

In resource tree:

![image](https://user-images.githubusercontent.com/628140/72372885-855d3b80-3707-11ea-8f02-e7810d3b04c6.png)

In list view:

![image](https://user-images.githubusercontent.com/628140/72373168-06b4ce00-3708-11ea-906b-6f9b5635d50c.png)

**Version**

```shell
argocd: v1.4.0-rc1+5af52f6
  BuildDate: 2020-01-13T17:19:43Z
  GitCommit: 5af52f66988ad8fa0d6b977d7f5aedcdb9f5a521
  GitTreeState: clean
  GoVersion: go1.12.6
  Compiler: gc
  Platform: linux/amd64
argocd-server: v1.4.0-rc1+5af52f6
  BuildDate: 2020-01-13T17:22:26Z
  GitCommit: 5af52f66988ad8fa0d6b977d7f5aedcdb9f5a521
  GitTreeState: clean
  GoVersion: go1.12.6
  Compiler: gc
  Platform: linux/amd64
  Ksonnet Version: v0.13.1
  Kustomize Version: Version: {Version:kustomize/v3.2.1 GitCommit:d89b448c745937f0cf1936162f26a5aac688f840 BuildDate:2019-09-27T00:10:52Z GoOs:linux GoArch:amd64}
  Helm Version: v2.15.2
  Kubectl Version: v1.14.0
```

**Logs**

N/A

**Comment by alexmt** [comment]
thank you for bug report @olivierlemasle ! https://github.com/argoproj/argo-cd/pull/2978

**Comment by olivierlemasle** [comment]
and thank you @alexmt for the bug fix!

**Comment by olivierlemasle** [comment]
Closed with #2978
